### DISCUSSION TITLE
Fluentbit Retry_Limit with storage

### DISCUSSION THREAD

**Discussion by GevorgTonoyan** [fluent/fluent-bit#9542] [Q&A] [answered]
_If the return status was OK, it means it was successfully able to process and flush the data. If it returned an Error status, it means that an unrecoverable error happened and the engine should not try to flush that data again._

In a scenario where I have a **retry_limit**  set to N and fluentbit fails to flush the chunk does it mean that the log is lost? I am using filesystem storage.

**Comment by patrick-stephens** [comment, marked as answer]
Yes, after N tries it will not try anymore to send that data and move on to the next chunk.
